### ISSUE TITLE
New name for "windows-gaming-driver"

### ISSUE THREAD

**Issue by oberien** [main--/windows-gaming#66]
Currently the host side of our software containing the driver and setup is called `windows-gaming-driver`. As it was first planned to only contain the actual diver, we should think about a new name for both the folder and the cargo package name. Here are some of my horrible suggestions:

* host-agent
* host
* windows-gaming

**Comment by main--** [comment]
I agree that the name is bad. But I'm fairly certain it needs to include "windows-gaming" (our project name - although we might even decide to change that) but not match it exactly (as the project consists of multiple pieces of software).

idk naming is hard???

**Comment by oberien** [comment]
Suggestions:
* Hyper-G
* gemu
* libgame

**Comment by main--** [comment]
Top contender: "dumbledoor" (by @moritzuehling)

**Comment by oberien** [comment]
Are we dumbledoor yet?

**Comment by oberien** [comment]
Are we dembledoor yet?

**Comment by oberien** [comment]
Are we dumbledoor yet?

**Comment by oberien** [comment]
Are we dumbledoor yet?

**Comment by moritzuehling** [comment]
So, what about dumbledoor now? The name has quite grown on me, to be honest.

**Comment by oberien** [comment]
[There are already projects out in the wild which assume our name to become dumbledoor.](https://github.com/moritzuehling/dumbledoor-config)
